packages/recomponents/package.json

```diff
@@ -8,12 +8,13 @@
   "license": "MIT",
   "scripts": {
     "test": "vue-cli-service test:unit",
-    "lint": "npm run lint:js && npm run lint:styles",
+    "lint": "run-s lint:js lint:styles",
     "lint:js": "vue-cli-service lint",
     "lint:styles": "stylelint **/*.scss",
-    "build": "npm run build:wc && npm run build:lib",
-    "build:wc": "vue-cli-service build --target wc-async --name recomponents 'src/**/*.vue'",
-    "build:lib": "vue-cli-service build --target lib --name recomponents 'src/index.js'",
+    "build": "run-s clean:dist build:wc build:lib",
+    "build:wc": "vue-cli-service build --target wc-async --name recomponents 'src/**/*.vue' --no-clean",
+    "build:lib": "vue-cli-service build --target lib --name recomponents 'src/index.js' --no-clean",
```

Comment:
Why would we want to keep the dist folder? It can potentially create a lot of issues :|

Reply:
If we omit the `--no-clean` flag and run the `build` script it clears the dist folder after every step. This way we purge our artifacts from `build:wc` when we start the `build:lib` test and so we could never ship the WebComponent artifacts.

Instead, I have added a new script (`clean:dist`) which is executed in the beginning of `build` and cleans out the dist folder every time we need to create a artifacts for deployment :)

Reply:
Btw, I completely agree that not ever purging the dist folder can lead to many issues :D 
I think that it would suffice purging it before `yarn build` though, instead of purging after every build step (that would require two separate dist folders, one for each build step)

Reply:
Oh crap, I didn't see that. Sorry, my bad. But it seems it wasn't working properly before? Anyway - good job on that!